c2pa/claim-templates/claim_create.json

```diff
@@ -0,0 +1,88 @@
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "{{author_type}}",
+                        "credential": [
+                        ],
+                        "identifier": "{{author_identifier}}",
+                        "name": "{{author_name}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "c2pa.actions",
+            "data": {
+                "actions": [
+                    {
+                        "action": "c2pa.created",
+                        "when": "{{authenticated_asset_created_timestamp}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "stds.exif",
+            "data": {
+                "@context": {
+                    "exif": "http://ns.adobe.com/exif/1.0/"
+                },
+                "exif:GPSLatitude": "{{gps_latitude}}",
+                "exif:GPSLatitudeRef": "{{gps_latitude_ref}}",
+                "exif:GPSLongitude": "{{gps_longitude}}",
+                "exif:GPSLongitudeRef": "{{gps_longitude_ref}}",
+                "exif:GPSTimeStamp": "{{gps_timestamp}}"
+            }
+        },
+        {
+            "label": "stds.iptc.photo-metadata",
+            "data": {
+                "@context": {
+                    "Iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/2008-02-29/",
+                    "dc" : "http://purl.org/dc/elements/1.1/"
+                },
+                "dc:creator": [
+                    "{{author_name}}"
+                ],
+                "dc:rights": "{{copyright}}",
+                "Iptc4xmpExt:LocationCreated": {
+                    "Iptc4xmpExt:CountryCode": "{{location_country_code}}",
+                    "Iptc4xmpExt:CountryName": "{{location_country_name}}",
+                    "Iptc4xmpExt:ProviceState": "{{location_province_state}}",
+                    "Iptc4xmpExt:City": "{{location_city}}",
+                    "Iptc4xmpExt:Sublocation": "{{location_sublocation}}",
+                    "Iptc4xmpExt:LocationDetails": "{{location_location_details}}"
+                }
+            }
+        },
+        {
+            "label": "org.starlinglab.integrity",
+            "data": {
+                "starling:identifier": "{{starling_identifier}}",
+                "starling:signatures": [
+                    {
+                        "starling:provider": "{{signature_provider}}",
+                        "starling:algorithm": "{{signature_algorithm}}",
+                        "starling:publicKey": "{{signature_public_key}}",
+                        "starling:signature": "{{signature_signature}}",
+                        "starling:authenticatedMessage": "{{signature_authenticated_message}}",
+                        "starling:authenticatedMessageDescription": "{{signature_authenticated_message_description}}",
+                        "starling:authenticatedMessagePublic": {
```

Comment:
This is a subset of data that is exposed publicly, inside the signed bundle. This is bc the original bundle contains information that may be internal and should not be disclosed publicly (e.g. precise location, in some cases).